### PR TITLE
[Doc Improvement][New RSC permission for Teams app installation][3885389]

### PR DESCRIPTION
**Pull request by v-sdhakshina**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ba6bad71df903e060dffd0f6c4da3c547dfb4991/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 94 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c2ca9062-c868-43f5-8c13-0d94569e8ea6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/graph-api/rsc/resource-specific-consent.md [hunk 1]
```diff
@@ -169,6 +169,7 @@ The following table provides RSC permissions for a user and their applicable dat
 | `MeetingParticipantReaction.Read.User` | Read the user's reactions while participating in a meeting. |Supported |NA |
 | `OutgoingVideoStream.Write.User` | Modify the user's outgoing video. |Supported |NA |
 | `TeamsActivity.Send.User` | Send activity notifications to the user. |NA |Supported |
+| `TeamsAppInstallation.Read.User` |  Read the user's action on the Teams app installation, upgrade, or delete event. | NA | Supported |
```
Comment (AdityaSharma6) [inline msteams-platform/graph-api/rsc/resource-specific-consent.md:172]: Reads the Teams Apps for a user.

**Review by AdityaSharma6 — CHANGES_REQUESTED** [review]

File: msteams-platform/graph-api/rsc/resource-specific-consent.md (hunk 1 above)
Comment (prshinde-msft) [reply msteams-platform/graph-api/rsc/resource-specific-consent.md:172]: I think let's stick to the same format we have for the corresponding .chat and .group permissions.
"Read the apps that are installed in the user's personal scope."

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/744823b00e7c07cdf077449bcc0bb3fcb213e637/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 94 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/94c18de4-c257-466f-86e7-06871b4fabb4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8107817d95be17de40ce34185907a8c7b0885fac/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 94 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/117785d4-43de-48db-8502-ea2f378ca5fa) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by AdityaSharma6 — APPROVED** [review]
Looks good to me!

**Review by JomyJohn — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Review by v-deabira — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/50a3c9ef19e9d3c8645f60830cc55b89fe4e7531/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 93 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3e602f97-3ccf-41f8-845c-3b4a592f9c25) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-deabira — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/38cd46e5984e87eddab749de8f260cfc21cb4957/msteams-platform%2Fgraph-api%2Frsc%2Fresource-specific-consent.md) | 93 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/47800809-9ae3-4d23-bd86-dd2a3adec64b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**